### ISSUE TITLE
g_object_unref: assertion 'G_IS_OBJECT (object)' failed

### ISSUE THREAD

**Issue by SpineEyE** [acrisci/playerctl#78]
Hi, when playerctl doesn't find a player, I get an error like the following one:

```
(playerctl:5764): GLib-GObject-CRITICAL **: 17:53:01.398: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
```

I'm on Arch Linux, Gnome, using playerctl v0.6.0.

**Comment by acrisci** [comment]
Yeah I can reproduce that.

**Comment by acrisci** [comment]
Fixed in 2032a74
